From 26e00e62300b1cc8f714ea11fc3856805652b67e Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Fri, 7 Jan 2011 10:46:43 +0800
Subject: [PATCH] ENGR00137718-2 imx28 : add system revision for imx28

Add the system revision number for imx28.
Also update the copyright.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 arch/arm/mach-mx28/mx28evk.c |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-mx28/mx28evk.c b/arch/arm/mach-mx28/mx28evk.c
index 308cace..97ce49c 100644
--- a/arch/arm/mach-mx28/mx28evk.c
+++ b/arch/arm/mach-mx28/mx28evk.c
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2009-2010 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright (C) 2009-2011 Freescale Semiconductor, Inc. All Rights Reserved.
  *
  * This program is free software; you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
@@ -146,6 +146,9 @@ static void __init mx28evk_init_machine(void)
 	mx28evk_pins_init();
 	mx28_device_init();
 	mx28evk_device_init();
+
+	/* init the system revesion */
+	system_rev = 0x28011;
 }
 
 MACHINE_START(MX28EVK, "Freescale MX28EVK board")
-- 
1.5.4.4

